File: core/src/com/unciv/models/gamebasics/Policy.java
package com.unciv.models.gamebasics;

import com.unciv.models.linq.Linq;
import com.unciv.models.stats.INamed;

/**
 * Created by LENOVO on 12/22/2017.
 */

public class Policy implements INamed{
    public String name;
    public String description;
    public String branch;
    public int row;
    public int column;
    public Linq<String> requires;

    @Override
    public String getName() {
        return name;
    }
}

File: core/src/com/unciv/models/gamebasics/Policy.kt
package com.unciv.models.gamebasics

import com.unciv.models.stats.INamed

open class Policy : INamed {
    lateinit var branch: PolicyBranch // not in json - added in gameBasics

    override lateinit var name: String
    lateinit var description: String
    var row: Int = 0
    var column: Int = 0
    var requires: ArrayList<String>? = null

    override fun toString(): String {
        return name
    }
}

